Makefile

```diff
@@ -0,0 +1,62 @@
+.PHONY: setup
```

Comment:
Such a makefile is the state of the art in the python world?

Reply:
At a first glance, I would rather go for GitHub Actions and for the local run using Act (https://github.com/nektos/act).

Reply:
As discussed yes, it is however only here to make local development much easier, e.g. installing, freezing, linting, etc, because the python universe has too many tools.